synapse/models/inet.py

```diff
@@ -29,6 +29,7 @@ def getDataModel():
 
             ('inet:cidr4',   {'ctor':'synapse.models.inet.CidrType','doc':'An IPv4 CIDR type','ex':'1.2.3.0/24'}),
 
+            ('inet:urlfile', {'subof':'comp','fields':'file,file:bytes|url,inet:url','doc':'A File at a Universal Resource Locator (URL)'}),
```

Comment:
lets swap the order of file/url ( there's an intuitive causality probably related to old "directed edge" concept where "following the url results in the file" ).